tsconfig.json

```diff
@@ -38,7 +38,13 @@
       "bazel-out/macos_arm64-fastbuild/bin",
       "bazel-out/macos_arm64-opt/bin",
       "bazel-out/macos_x86_64-fastbuild/bin",
-      "bazel-out/macos_x86_64-opt/bin"
+      "bazel-out/macos_x86_64-opt/bin",
+      "bazel-out/platform_linux_arm64-fastbuild/bin",
+      "bazel-out/platform_linux_arm64-opt/bin",
+      "bazel-out/platform_linux-fastbuild/bin",
+      "bazel-out/platform_linux-opt/bin",
+      "bazel-out/platform_linux_x86_64-fastbuild/bin",
+      "bazel-out/platform_linux_x86_64-opt/bin"
```

Comment:
Why do we need both `platform_linux-fastbuild` and `platform_linux_x86_64-fastbuild` here?

Reply:
this thing needs directories for every platform string or the build fails when you build with that platform. So long as both `platform_linux` and `platform_linux_x86_64` exist as platforms, we need both.